src/iact_estimator/scripts/estimator.py

```diff
@@ -101,7 +113,10 @@ def main():
             label=source_name,
         )
 
-    energy_bins, gamma_rate, background_rate = prepare_data(config)
+    if not performance_data:
+        energy_bins, gamma_rate, background_rate = prepare_data(config)
+    else:
+        energy_bins, gamma_rate, background_rate = prepare_data(performance_data)
```

Comment:
Here we need to put both of `config` and `performance_data`

Reply:
fixed